### DISCUSSION TITLE
How do I uninstall when I built from source?

### DISCUSSION THREAD

**Discussion by Android789515** [wezterm/wezterm#4799] [Q&A] [answered]
### What Operating System(s) are you running on?

Linux X11

### Which Wayland compositor or X11 Window manager(s) are you using?

i3wm with picom

### WezTerm version

20230904-133103-914f18b1

### Ask your question!

I built the program from source but I am removing it now.  I don't know where it is.

**Comment by wez** [comment]
there are no prescribed steps to follow to actually install the software when building from source. You need to undo whatever you did when you installed it, and I don't know what you did :-p

I recommend using native packages where possible so that you don't have to think about this so much in the future.

**Reply by Android789515** [reply]
I think I used cargo to build it but I'm not sure where it puts it.

**Reply by wez** [reply]
cargo is not a supported way to install wezterm. You'll need to figure out what you did and undo it. I'm sorry, but I cannot help you with this.

**Reply by Android789515** [reply]
It's right here:
https://wezfurlong.org/wezterm/install/source.html#installing-from-source

**Reply by Android789515** [reply, marked as answer]
I found it in my binaries.  Figured I had to move it somewhere logical.
